packages/core/lib/debug/interpreter.js

```diff
@@ -738,6 +750,11 @@ class DebugInterpreter {
   updatePrintouts(userArgs, selections, printOuts) {
     let tempPrintouts = new Set();
     for (let argument of userArgs) {
+      // skip line number options
+      if (((argument[0] === "-") || (argument[0] === "+")) &&
```

Comment:
OK, this part I think has some style problems.

Firstly, I would strongly suggest that you not use multi-line conditionals (or other blocks) without braces.  These can be something of a hazard.  If you have a conditional, either it should use braces, or it should fit on a single line.

Secondly, there's some repetition here with this check on the following lines.  I'd like to suggest that this be consolidated somehow.

Reply:
Actually, hold on -- on looking at this some more... do we even need to explicitly skip these?  It seems to me that the remaining code here will ignore any numbers put in just fine.  I'm not sure we need an explicit skip here; I suspect this addition can just be deleted.  Am I missing something?  (Of course if you do delete it, it's probably worth putting in a comment about this. :) )

Reply:
You are right, no need to skip these since the remaining code will ignore any numbers. The skip line code will be deleted.

Reply:
OK!